src/resources/token.js

```diff
@@ -54,6 +58,62 @@ class Token extends ResourceBase {
     await this.getContract()
     return await this.contract.methods.paused().call()
   }
+
+  /**
+   * Relays calls to the contract service
+   */
+  async call(methodName, args, opts) {
```

Comment:
Why does this not call the `contractService` method directly?

Reply:
Ah, I see, it needs the `contractName`. ✅

Reply:
Right on - it's modeled after the marketplace adapter here:
https://github.com/OriginProtocol/origin-js/blob/john/boost_approval/src/adapters/marketplace/v00.js#L26-L33